order-service/app.py

```diff
@@ -1,76 +1,81 @@
 
-    return jsonify({'message': 'Order placed successfully', 'order_id': cursor.lastrowid})
+        # Create the new order
+        new_order = Order(
+            user_id=user.id,
+            product_id=product.id,
+            quantity=data['quantity'],
+            total_price=data['total_price'],
+        )
+        session.add(new_order)
+        session.commit()
+        return jsonify({'message': 'Order placed successfully', 'order_id': new_order.id})
+    except Exception as e:
+        session.rollback()
+        return jsonify({'error': str(e)}), 500
+    finally:
+        session.close()
 
 @app.route('/orders/payment', methods=['POST'])
 def handle_payment():
     data = request.get_json()
-    cursor.execute("UPDATE orders SET paid=1 WHERE id=%s", (data['order_id'],))
-    db.commit()
-    return jsonify({'message': 'Payment processed successfully'})
+    session = SessionLocal()
+    try:
+        order = session.query(Order).filter(Order.id == data['order_id']).first()
+        if not order:
+            return jsonify({'message': 'Order not found'}), 404
+
+        order.paid = True
+        session.commit()
+        return jsonify({'message': 'Payment processed successfully'})
+    except Exception as e:
+        session.rollback()
+        return jsonify({'error': str(e)}), 500
+    finally:
+        session.close()
 
 @app.route('/orders/status/<int:order_id>', methods=['GET'])
 def get_order_status(order_id):
-    cursor.execute("""
-        SELECT orders.id, 
-               users.email as user_email, 
-               products.name as product_name,
-               orders.quantity, 
-               orders.total_price, 
-               orders.paid 
-        FROM orders
-        JOIN products ON orders.product_id = products.id
-        JOIN users ON orders.user_id = users.id
-        WHERE orders.id = %s
-    """, (order_id,))
-    order = cursor.fetchone()
-    
-    if order:
-        return jsonify({
-            'id': order[0],
-            'user_email': order[1],
-            'product_name': order[2],
-            'quantity': order[3],
-            'total_price': order[4],
-            'paid': order[5]
-        })
-    else:
-        return jsonify({'message': 'Order not found'}), 404
+    session = SessionLocal()
+    try:
+        order = session.query(Order).filter(Order.id == order_id).first()
+        if order:
+            return jsonify({
+                'id': order.id,
+                'user_email': order.user.email,
+                'product_name': order.product.name,
+                'quantity': order.quantity,
+                'total_price': str(order.total_price),
+                'paid': order.paid
+            })
+        else:
+            return jsonify({'message': 'Order not found'}), 404
+    finally:
+        session.close()
 
 if __name__ == '__main__':
-    app.run()
\ No newline at end of file
+    app.run(host='0.0.0.0', port=5000)
```

Comment:
Remove the args, they are declared in the docker file

Reply:
Okay